GetStream/stream-chat-android
# [CAS-885] Fix sample app navigation after clicking on notification

https://stream-io.atlassian.net/browse/CAS-885

### 🎯 Goal

Fix navigation after clicking on notification and fix notification's reply action

### 🛠 Implementation details

1. Fixed notification's reply action by passing the user to send message request
2. Added `Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK` flags to intent which starts `HostActivity` after clicking on notification
3. Remove unnecessary `StartupFragment`

### 🧪 Testing
1. Put an app into the background
2. Send message from another device
3. Click the notification
4. Kill the app
5. Send a message from another device
6. Click the notification
7. In both cases you should be redirected directly to channel view

### ☑️ Checklist


### 🎉 GIF
![giphy](https://user-images.githubusercontent.com/17440581/119147424-ddb6f280-ba4b-11eb-952e-bbd2a9e6fc56.gif)
